VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTopic.java

```diff
@@ -106,4 +115,12 @@ private void appendTopicIds(List<String> ids) {
             child.appendTopicIds(ids);
         }
     }
+
+    public boolean isAllType() {
+        return DiscussionTopic.ALL_TOPICS_ID.equals(identifier);
+    }
+
+    public boolean isFollowingType() {
```

Comment:
on a side note, I thought about the naming for quite some time and resorted on these cuz I didn't wanted what happened here:
<img src="http://www.commitstrip.com/wp-content/uploads/2015/10/Strip-Trouver-le-nom-de-variable-english650-final.jpg" />